### ISSUE TITLE
Taint mode

### ISSUE THREAD

**Issue by pink-mist** [LorenzoTa/step-by-step-tutorial-on-perl-module-creation-with-tests-and-git#6]
There are lots of problems with using taint mode, and these days there are much better and nicer ways to validate user-supplied data and prevent environment poisoning than using ```-T```. I'd suggest rewording the sections mentioning ```-T```.

**Comment by LorenzoTa** [comment]
thanks,
I'm very ignorant  about this.. 

[the relevant part](https://github.com/LorenzoTa/step-by-step-tutorial-on-perl-module-creation-with-tests-and-git/blob/master/source/tutorial-english.html#L608) Is there only because I acted in a wrong way during the creation of the module and the relative online repository.

I realized it after many commits and was too late to revert.

Security can be whole chapter per se, but I have no knoweledge about it
